lib/Sema/TypeCheckConcurrency.cpp

```diff
@@ -2782,13 +3222,19 @@ ActorIsolation ActorIsolationRequest::evaluate(
     // inferred, so that (e.g.) it will be printed and serialized.
     ASTContext &ctx = value->getASTContext();
     switch (inferred) {
-    case ActorIsolation::Independent:
+    case ActorIsolation::Independent: {
       if (onlyGlobal)
         return ActorIsolation::forUnspecified();
 
-      value->getAttrs().add(new (ctx) ActorIndependentAttr(
-                              ActorIndependentKind::Safe, /*IsImplicit=*/true));
+      auto var = dyn_cast<VarDecl>(value);
+      if ((var && !var->isLet()) || !var) {
+        // It is illegal to add @actorIndependent to let properties,
+        // and if we did this would fail in type checking attributes.
+        value->getAttrs().add(new(ctx) ActorIndependentAttr(
```

Comment:
`@actorIndependent` is going away. I don't think we need this.

Reply:
removed, thanks